### ISSUE TITLE
Image Loading

### ISSUE THREAD

**Issue by tibuch** [haesleinhuepf/devbio-napari#5]
Hello,

Very nice bundle. Just tried to install it and everything seems to be working. However, I noticed that Metamorph (.stk) files are not loaded correctly. Would it be possible to add `napari-aicsimageio`?

These installation instructions worked for me:
```
conda install -c conda-forge openjdk
pip install aicsimageio bioformats_jar
pip install napari-aicsimageio
````

**Comment by haesleinhuepf** [comment]
Hi @tibuch ,

yes, I would love to add it again. It [was just recently removed](https://github.com/haesleinhuepf/devbio-napari/commit/fa6695c0ff56ab432b3404f89677b56d5159f14a) because of licensing issues (it's now GPL licensed). I'm thinking of starting `devbio-napari-gpl` where we could collect those things and also for example napari-pymeshlab (which is also GPL licensed).

Let me know what you think!

Best,
Robert

**Comment by tibuch** [comment]
Makes sense. I think `devbio-napari-gpl` would then be helpful. Installing two things (`devbio-napari` and `devbio-napari-gpl`) is still better than installing everything individually :)

Thanks for the work!

Best,
Tim-Oliver

**Comment by haesleinhuepf** [comment]
Let's keep this one open to not forget to do it 😉

**Comment by evamaxfield** [comment]
Just wanted to chime in and say that I have also been considering this behavior as well... creating a `napari-aicsimageio` (for BSD) and a `napari-aicsimageio-gpl` (for GPL)

Licensing is a nightmare and the fact that we need to have two packages for it is so annoying and just creates a worse user-experience but 🤷‍♀️ the lawyers tell me I need to.
